containers/runtime/driver/run.sh

```diff
@@ -24,7 +24,9 @@ fi
 
 /src/code/bazel-bin/test/cpp/qps/qps_json_driver --quit=true
 
-if [ "$BQ_RESULT_TABLE" != "" ]
-then
+if [ "$BQ_RESULT_TABLE" != "" ]; then
```

Comment:
This is fine. You could also use `-a` to put both conditions in one test.

Reply:
I have updated the two conditions to be in the same statement. Thank you.